app/services/create_judgement.rb

```diff
@@ -0,0 +1,37 @@
+class CreateJudgement
+
+  def self.call(evaluation, params)
+    return false unless self.changed?(evaluation, params)
+    message = self.create_message(evaluation, params)
+    EvaluationMessage.create(sent_at: DateTime.now, message: message, from_system: true,
+      semester: evaluation.semester, sender_user: nil, group: evaluation.group)
+    evaluation.update(params)
+  end
+
+  def self.create_message(evaluation, params)
+    old_point_request_status = evaluation.point_request_status
+    new_point_request_status = params[:point_request_status]
+    if old_point_request_status == new_point_request_status
+      point_request_change = 'nem változott'
+    else
+      point_request_change = "#{old_point_request_status} -> #{new_point_request_status}"
+    end
+
+    old_entry_request_status = evaluation.entry_request_status
```

Comment:
This part could be organized to a new function as it contains a considerable amount of duplicated code. Basically the only difference between the two is the `:point_request_status` key that could easily be a parameter.